org.eclipse.zest.layouts/src/org/eclipse/zest/layouts/algorithms/TreeLayoutAlgorithm.java

```diff
@@ -583,25 +560,28 @@ private int getLevelRecursive(InternalNode layoutEntity, int i, Set seen, Intern
 	 * @param nodeToFind
 	 * @return
 	 */
-	private int indexOfInternalNode(InternalNode[] nodes, InternalNode nodeToFind) {
+	private static int indexOfInternalNode(InternalNode[] nodes, InternalNode nodeToFind) {
 		for (int i = 0; i < nodes.length; i++) {
 			InternalNode node = nodes[i];
 			if (node.equals(nodeToFind)) {
 				return i;
 			}
 		}
-		throw new RuntimeException("Couldn't find index of internal node: " + nodeToFind);
+		throw new RuntimeException("Couldn't find index of internal node: " + nodeToFind); //$NON-NLS-1$
 	}
 
 	@Override
 	protected boolean isValidConfiguration(boolean asynchronous, boolean continueous) {
 		if (asynchronous && continueous) {
 			return false;
-		} else if (asynchronous && !continueous) {
+		}
+		if (asynchronous && !continueous) {
 			return true;
-		} else if (!asynchronous && continueous) {
+		}
+		if (!asynchronous && continueous) {
 			return false;
-		} else if (!asynchronous && !continueous) {
+		}
+		if (!asynchronous && !continueous) {
 			return true;
 		}
 
```

Comment:
Same as before, that should be equivalent to `return !continueous;`

Reply:
I found this code extra spooky and was to afraid to change it. But you are totally right. Changed it accordingly.